### PR TITLE
ADD multi company security rule for package preparation

### PR DESCRIPTION
**Pull request by eLBati**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

**Comment by tafaRU** [comment]
:+1:

File: stock_picking_package_preparation/security/package_preparation_security.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data noupdate="1">
+
+    <record model="ir.rule" id="pack_preparation_comp_rule">
+        <field name="name">Package Preparation multi-company</field>
+        <field name="model_id" ref="model_stock_picking_package_preparation"/>
+        <field name="global" eval="True"/>
+        <field name="domain_force">['|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]</field>
```
Comment (lmignon) [inline stock_picking_package_preparation/security/package_preparation_security.xml:8]: Is `('company_id','=',user.company_id.id)` included by the child_of operator?

Comment (Garamotte) [inline stock_picking_package_preparation/security/package_preparation_security.xml:8]: If I'm not mistaken, it's included.

**Comment by Garamotte** [comment]
:+1:

File: stock_picking_package_preparation/security/package_preparation_security.xml (hunk 1 above)
Comment (guewen) [inline stock_picking_package_preparation/security/package_preparation_security.xml:8]: Yes it is

File: stock_picking_package_preparation/security/package_preparation_security.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data noupdate="1">
+
+    <record model="ir.rule" id="pack_preparation_comp_rule">
+        <field name="name">Package Preparation multi-company</field>
+        <field name="model_id" ref="model_stock_picking_package_preparation"/>
+        <field name="global" eval="True"/>
```
Comment (guewen) [inline stock_picking_package_preparation/security/package_preparation_security.xml]: You shouldn't write to `global`, this is a function field which is true when the rule has no group.

**Comment by archetipo** [comment]
:+1:

**Comment by eLBati** [comment]
@guewen , thanks, removed `global`

File: stock_picking_package_preparation/security/package_preparation_security.xml (hunk 1 above)
Comment (lmignon) [inline stock_picking_package_preparation/security/package_preparation_security.xml:8]: @sylvain-garancher, @guewen Thank you for the information.

**Pull request merged**